apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -4,57 +4,35 @@
 import os
 import random
 from copy import deepcopy
-from enum import Enum
 from typing import TypedDict
 
 import chevron
 import lxml.html
 import prairielearn as pl
 from dag_checker import grade_dag, lcs_partial_credit, solve_dag
-from lxml.etree import _Comment
+from order_blocks_options_parsing import (
```

Comment:
If you want to save space, you can do `import order_blocks_options_parsing as op` and then all of these can be done with an `op.` prefix